tar/storage/storage_read.c

```diff
@@ -249,6 +249,35 @@ storage_read_add_name_cache(STORAGE_R * S, char class, const uint8_t name[32])
 	return (-1);
 }
 
+/**
```

Comment:
It's going to be moved to a new file `storage_read_cache.c` file.  The planned history is:
- this PR: make it an independent function
- next PR (#505): make it accept a new `struct storage_read_cache * cache` instead of `STORAGE_R * S`
- following PR (not online yet): move all cache functions (including this one) into the new `storage_read_cache.c`